audobject/core/config.py

```diff
@@ -3,6 +3,10 @@
 
 class config:
 
+    PACKAGE_MISMATCH_WARN_LEVEL = define.PackageMismatchWarnLevel.STANDARD
```

Comment:
At the moment the configuration entry looks a little bit empty:

![image](https://user-images.githubusercontent.com/173624/158966946-1dbdee41-e121-4a70-a20b-abf4666c93d3.png)

compared e.g. with https://audeering.github.io/audb/api.html#config

So maybe we add a docstring as well:

```suggestion
    r"""Get/set configuration values for the :mod:`audobject` module.
    
    Example:
        
        >>> config.PACKAGE_MISMATCH_WARN_LEVEL
        1
        >>> config.PACKAGE_MISMATCH_WARN_LEVEL = 0
        >>> config.PACKAGE_MISMATCH_WARN_LEVEL
        0
        
    """
    
    PACKAGE_MISMATCH_WARN_LEVEL = define.PackageMismatchWarnLevel.STANDARD
```